### PR TITLE
AllMySms notifier bridge

### PR DESCRIPTION
**Pull request by qdequippe**
| Q             | A
| ------------- | ---
| Branch?       | master (new feature)
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       |
| License       | MIT
| Doc PR        |

Add [AllMySMS](https://www.allmysms.com) (French SMS provider) Bridge to Notifier component.

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/AllMySMS/composer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "name": "qdequippe/allmysms-notifier",
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/AllMySMS/composer.json]: Shouldn’t the name be changed?

**Comment by OskarStark** [comment]
Please keep an eye on https://github.com/symfony/symfony/pull/33967

File: src/Symfony/Component/Notifier/Bridge/AllMySMS/composer.json (hunk 1 above)
Comment (qdequippe) [reply src/Symfony/Component/Notifier/Bridge/AllMySMS/composer.json]: yes you're right!

File: src/Symfony/Component/Notifier/Bridge/AllMySMS/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+AllMySMS Notifier
+===============
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/AllMySMS/README.md]:
```suggestion
=================
```

File: src/Symfony/Component/Notifier/Bridge/AllMySMS/AllMySMSTransport.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                'apiKey' => $this->apiKey,
+                'tpoa' => $this->tpoa,
+                'mobile' => $message->getPhone(),
+                'message' => $message->getSubject(),
+            ],
+        ]);
+
+        $result = $response->toArray();
+
+        if (100 != $result['status']) {
```
Comment (ismail1432) [inline src/Symfony/Component/Notifier/Bridge/AllMySMS/AllMySMSTransport.php]: you can't check the successfull code status ?

Comment (qdequippe) [reply src/Symfony/Component/Notifier/Bridge/AllMySMS/AllMySMSTransport.php]: You're right thx! in fact the provider developed a new REST API so I will use it

**Comment by qdequippe** [comment]
Hi @OskarStark did you recommend me to close this PR and create my own integration in a different repo?

**Pull request closed without merging**